npc/re/merchants/rgsr_in.txt

```diff
@@ -0,0 +1,451 @@
+			close2;
+			callshop("barter_rgsr_in_3");
+			end;
+		case 4:
+			mes "[Grid]";
+			mes "If you need anything, come again.";
+			close;
+		}
+		end;
+	case 2:
+		mes "[Grid]";
+		mes "As we see in our research, the materials are very rare. There are adventurers hired by our team, but the material is not comparable to the amount we need.";
+		mes "So when the adventurers came with the materials, they decided to provide the corresponding research value.";
+		next;
+		mes "[Grid]";
+		mes "Luckily, we do the research and adventurers who come to us provide materials to use our technology.";
+		mes "If you want to modify you only need to bring the materials presented.";
+		next;
+		mes "[Grid]";
+		mes "You're an adventurer, so you know what that means?";
+		close;
+	case 3:
+		mes "[Grid]";
+		mes "If you need anything, come again.";
+		close;
+	}
+}
+
+rgsr_in,137,178,3	script	Accessories Researcher#rg02	MD_ED_M_SCIENCE,{
+	mes "[Champ]";
+	mes "Finally, I talked an adventurer and asked him to confirm his research, is that you?";
+	mes "Unfortunately, our team hasn't done enough to get out there yet.";
+	next;
+	mes "[Champ]";
+	mes "It's still incomplete.";
+	mes "Next time the colon permit falls, we will see then.";
+	close;
+
+OnInit:
+	setunittitle( getnpcid (0), "Team Zertar");
+	end;
+}
+
+rgsr_in,136,171,5	script	Special Equipment Researcher#rg02	MD_ED_M_SCIENCE,{
+	mes "[Stack]";
+	mes "See Team Licht's performance?";
+	mes "We're proud of this lab.";
+	mes "Which one do you want to modify?";
+	mes "Modifiable items are located in <TIPBOX>[List]<INFO>9166</INFO></TIPBOX>.";
+	next;
+	switch( select( "Non-standard armor", "Non-standard weapon", "Quit" ) ) {
+	case 1:
+		mes "[Stack]";
+		mes "^0000FFNon-standard armor^000000, there are lower, intermediate, and upper levels in the modification method.";
+		mes "Non-smelting armor is only capable of advanced modification.";
+		next;
+		switch( select( "Lower mods", "Intermediate mods", "Upper mods", "Quit" ) ) {
+		case 1:
+			mes "[Stack]";
+			mes "Armor Lower Mods? Okay.";
+			mes "^FF0000Smelting level is reduced by 5^000000, but consumes less material.";
+			mes "And for modification, the equipment must have a smelting level of +9 or more.";
+			close2;
+			item_reform("C_Armor_Reform_1");
+			end;
+		case 2:
+			mes "[Stack]";
+			mes "Armor Intermediate Mods? Okay.";
+			mes "^FF0000Smelting level is reduced by 1^000000, but consumes more material.";
+			mes "And for modification, the equipment must have a smelting level of +9 or more.";
+			close2;
+			item_reform("C_Armor_Reform_2");
+			end;
+		case 3:
+			mes "[Stack]";
+			mes "Armor Advanced Mods? Okay.";
+			mes "^FF0000Smelting level increases by 1^000000, but consumes more material.";
+			mes "And for modification, the equipment must have a smelting level of +9 or more.";
+			mes "^FF0000Non-smelting equipment^000000 can also be granted with advanced modifications, but the smelting level does not increase.";
```

Comment:
```suggestion
			mes "^FF0000Non-refineable equipment^000000 can also be granted with advanced modifications, but the refine level does not increase.";
```

Reply:
NOTE: We should check if I implemented item reformation like this in src. I dont remember checking the refineable flag.